.dockerignore

```diff
@@ -0,0 +1,8 @@
+kube
```

Comment:
"kube" feels a bit specific.

As far as I'm aware it sis considered bad practice to have that stuff in your repo anyway, so I'd suggest removing it.